source/sass/buyers-guide/views/catalog.scss

```diff
@@ -66,115 +66,93 @@ body {
   }
 
   .product-box-list-wrapper {
-    // Mixin for setting product box styling in product box list
```

Comment:
Since we don't want line 75-78 (negative margin) anymore and none of the variables `$num-of-gutter`, `$gutter-width`, `$row-total-gutter-width` get used anywhere, we don't need to have a mixin.